sklearn/preprocessing/_data.py

```diff
@@ -683,6 +683,11 @@ class StandardScaler(OneToOneFeatureMixin, TransformerMixin, BaseEstimator):
     than others, it might dominate the objective function and make the
     estimator unable to learn from other features correctly as expected.
 
+    StandardScaler is sensitive to outliers, and the features may scale
+    differently from each other in the presence of outliers. For an example
+    visualization, refer to the :ref:`standard_scaler_section` section in the
+    :ref:`sphx_glr_auto_examples_preprocessing_plot_all_scaling.py` example.
```

Comment:
```suggestion
    visualization, refer to :ref:`Compare StandardScaler with other scalers
    <standard_scaler_section>`.
```

don't need to have two links, one link would be enough.

Same with other links to sections.

Reply:
Sure, I have changed that. :)